src/List.js

```diff
@@ -28,10 +28,13 @@ const List = () => {
     const calculatedDate = calculateEstimate(
       lastPurchasedDate,
       itemSelected[0].frequency,
```

Comment:
Continuing off of my comments RE: `frequency`/`nextPurchase` above . . .
If `frequency` never reflects the current buying pattern, would it skew the `nextPurchase` estimate?

For example, if the user initially estimated a frequency of 7 days for an item, but because of their actual purchases the `nextPurchase` is 4 days, when another purchase is made `nextPurchase` is recalculated in reference to `frequency` which is still 7 days. Should it be recalculated with the current purchase pattern of 4 days instead?